packages/eslint-config-react-app/package.json

```diff
@@ -26,6 +27,6 @@
     "eslint-plugin-react-hooks": "1.x || 2.x"
   },
   "dependencies": {
-    "confusing-browser-globals": "^1.0.9"
+    "confusing-browser-globals": "^1.0.10"
```

Comment:
Mistake? we shouldn't edit any packages outside of `react-scripts`...

Reply:
Will double check but I hadn't done this myself will check upstream/master